src/limit-line-length/limit-line-length.js

```diff
@@ -8,6 +8,10 @@ function createCssRules(lineLengthLimit, isStickyHeaderEnabled) {
     }
 
     let cssRules = `
+        .refract-content-container, .comment-thread-container {
```

Comment:
`.refract-content-container` stops line from being above file comment thread

`.comment-thread-container` stops line from being within the padding of a line comment thread